doc/changes/changelog.md

```diff
@@ -1,5 +1,6 @@
 # Changes
 
+* [2.5.0](changes_2.5.0.md)
```

Comment:
This is just a bugfix, an increment of the patch version should be enough. Or do you plan other features?

Reply:
yea, I would release it end of this week and think until then we will get a feature done.